bcwallet/bcwallet.py

```diff
@@ -1269,8 +1275,11 @@ def cli():
             puts(colored.red("Invalid wallet entry: %s" % wallet))
 
     else:
-        puts("You've opened your HD wallet without specifying a master public or master private key, which you can do like this:\n")
-        print_bcwallet_basic_priv_opening(priv_to_display='xpriv123...')
+        puts("You've opened your HD wallet without specifying a master public or master private key.\n")
+        puts("To open your wallet with your master private key:\n")
+        print_bcwallet_basic_priv_opening(priv_to_display='xpriv123...\n')
+        puts("To open your wallet with your master public key:\n")
+        print_bcwallet_basic_priv_opening(priv_to_display='xpub123...\n')
```

Comment:
This is a lot of information, but I was trying to give more clarity on the public key mode. I would imagine most would want to open their wallet just to see their balance and transactions, more so than to do actual spending. Wouldn't using public key mode be better for this?

Personally, I would prefer to see a public watch only mode, without any transaction signing capabilities.